build-system/pr-check.js

```diff
@@ -126,7 +126,9 @@ function isBuildSystemFile(filePath) {
       !isFlagConfig(filePath) &&
       // Exclude visual diff files from build-system since we want it to trigger
       // visual diff tests.
-      !isVisualDiffFile(filePath);
+      !isVisualDiffFile(filePath))
+      // OWNERS.yaml files should trigger build system to run tests
+      || path.basename(filePath) === 'OWNERS.yaml';
```

Comment:
i want to specifically run this test https://github.com/ampproject/amphtml/pull/10342/files#diff-c7e49387b3bfb88dd86a9feddcc103c9R46 if any OWNERS.yaml file is changed

the other yaml files we don't need to worry about, this one is special as OWNERS.yaml should require an update to CODEOWNERS, i could also have done this as a pr-check.js but this sort of covers both unit test for the task as well as enforcement. 

let me know what you think. thanks!